packages/main/src/ColorPicker.js

```diff
@@ -472,11 +485,6 @@ class ColorPicker extends UI5Element {
 
 	_setValues() {
 		const hslColours = RGBToHSL(this._color);
```

Comment:
After removing lines 475-479, the following bug is present:
Change a number in some of the first three inputs(RGB inputs) and tab Tab. The new entered colour should be focused by the circle

Reply:
The described rows are now restored. Done.